pkg/devicetemplates/ttscsv.go

```diff
@@ -393,6 +438,33 @@ func (*ttsCSV) Convert(ctx context.Context, r io.Reader, ch chan<- *ttnpb.EndDev
 			EndDevice: dev,
 			FieldMask: ttnpb.FieldMask(paths...),
 		}
+
+		if t.profile != nil && t.profile.ShouldFetchProfile(dev) {
+			profileTmpl, err := t.profile.FetchProfile(ctx, dev)
+			if err != nil {
+				return err
+			}
+			// Set formatters to be form Device Repository.
```

Comment:
Drop this for now.

Reply:
Dropped, the following FieldMasks remain on this section:
```
"version_ids.vendor_id",
"version_ids.vendor_profile_id",
```
They are accounted for not when importing the end-device profile but when the vendor_id is provided through the request body ( a fallback is provided ).


Regarding the discussion of making the DR set the formatters instead of leaving it be handled on the converter. I'll look into how to make the change and I'll write an issue about it. Then we can go back and forward there if the approach is adequate. What you think? @adriansmares

Reply:
> Regarding the discussion of making the DR set the formatters instead of leaving it be handled on the converter. I'll look into how to make the change and I'll write an issue about it. Then we can go back and forward there if the approach is adequate. What you think? @adriansmares

This is fine with me.